corehq/apps/hqadmin/templates/hqadmin/partials/deploy_history.html

```diff
@@ -12,56 +12,25 @@ <h2 class="panel-title">
             <tr>
                 <th>Date</th>
                 <th>User</th>
-                <th>Branch</th>
-                <th>Last Commit</th>
-                <th></th>
+                <th>Diff url</th>
             </tr>
             </thead>
             <tbody>
             {% for deploy in deploy_history %}
-                {% with deploy.code_snapshot.commits|first as commit%}
                 <tr>
                     <td>
                         {{ deploy.date|naturaltime }}<br/>
                         <small>{{ deploy.date }}</small>
                     </td>
                     <td>{{ deploy.user }}</td>
-                    <td>{{ deploy.code_snapshot.current_branch }}</td>
                     <td>
-                        {% include "hqadmin/partials/commit.html" %}
```

Comment:
:+1:  by me. i don't trust this thing anyway since the deploy success task isn't all that reliable so I usually just ssh in if i really want to know the commit